m4_sessions/1806-inning/group6.md

```diff
@@ -1,33 +1,42 @@
 ## Group Members:
-
+Kelly Schroeder, Helen Dechat, Sage Vogt, David 'Doc' Daugherty
 
 
 ## Name of Session: 
-
+ VR & AR: How they will affect our lives in the future.
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+The ways in which AR and VR will affect future technologies and how those technologies will impact society regarding industry, healthcare, education, and entertainment. 
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+(What will students learn/leave with?): Excited AF about AR & VR. With a better understanding, and hopefully increase interest, in the coming wave of AR & VR technologies.
 
 ## Session Outline 
 
 (Please include who is facilitating and how much time you are going to spend in each section):
+     15 minutes on each of the 4 sections with 10 minute Q&A at the end.
+     Industry: Kelly
```

Comment:
Okay, I'd like to see all of your sections in a slide and instead of a Q&A you build out specific discussion questions.  This can get really boring to just learn a lot of new info.  Maybe you force them to take a side on a position or they could go to ONE of the categories they are super excited about and ONE that they know nothing about and then share in small mixed groups.